app/konnect/mesh-manager/migrate-zone.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Migrate a self-managed global zone control plane to {{site.konnect_saas}}
+content_type: tutorial
+---
+
+This guide explains how to migrate a self-managed global control plane to {{site.konnect_saas}}. This can be useful if you manage other services in {{site.konnect_saas}} that way you can view all your entities in one place. 
+
+## Prerequisites
+
+* A universal or Kubernetes cluster with a running zone control plane connected to the self-managed global control plane
+* [The latest version of kumactl](/mesh/latest/production/install-kumactl/) installed and configured to communicate with the zone control plane
+* A [{{site.mesh_product_name}} global control plane in {{site.konnect_short_name}}](/konnect/mesh-manager/service-mesh/#create-a-zone-in-the-global-control-plane)
+* [yq installed](https://github.com/mikefarah/yq)
+
+## Transfer resources from the self-managed global control plane to {{site.konnect_short_name}}
+
+1. Make sure the self-managed global control plane is active:
+  ```bash
+  kumactl config control-planes list
+  ```
+  This should return your self-managed global control plane. You can use port-forward of the On-Prem Global CP API port 5681 to localhost to get quick access with `kumactl`.
```

Comment:
I'm not sure what the second sentence means. Is there a command you can use to make sure the on prem global cp is active on port 5681? Or does this do something else?

Reply:
If a user runs self-managed global cp on Kubernetes then quite likely they might be not familiar how to operate it with `kumactl` (because they're using `kubectl`). Normally user has to authenticate `kumactl` with API by following the [instruction](https://docs.konghq.com/mesh/latest/production/secure-deployment/api-server-auth). But there is a simpler way and I wanted to add this like a tip – port-forward Global CP API port to localhost.

But I just noticed that the instruction I mentioned above already contains information about the "port-forward". So I think it's safe to remove the second sentence altogether.